pco/src/data_types/floats.rs

```diff
@@ -424,6 +424,29 @@ impl_float_number_like!(f16, u16, 1_u16 << 15, 9);
 mod tests {
   use super::*;
 
+  #[test]
+  fn test_choose_mult_mode() {
+    let base = 1.7;
+    let nums = (0..300).map(|i| (i as f64) * base).collect::<Vec<_>>();
+    let (mode, _) = choose_mode_and_split_latents(&nums, &ChunkConfig::default()).unwrap();
+    assert_eq!(
+      mode,
+      Mode::FloatMult(base.to_latent_ordered())
+    );
```

Comment:
It's a coincidence that we get exact equality when `base = 1.7`, right?  In general there may be rounding error?

Reply:
The heuristics are usually pretty good, but I changed it to 1.5 for more exact arithmetic.